src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/GenerateHearingBundleConfirmationTest.java

```diff
@@ -58,6 +67,39 @@ public void should_return_confirmation() {
         );
     }
 
+    @Test
+    public void should_return_notification_failed_confirmation() {
+
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(asylumCase);
+
+        when(callback.getEvent()).thenReturn(Event.GENERATE_HEARING_BUNDLE);
+        when(asylumCase.read(AsylumCaseFieldDefinition.HOME_OFFICE_HEARING_BUNDLE_READY_INSTRUCT_STATUS, String.class))
+            .thenReturn(Optional.of("FAIL"));
+
+        PostSubmitCallbackResponse callbackResponse =
+            generateHearingBundleConfirmation.handle(callback);
+
+        Assert.assertNotNull(callbackResponse);
+        Assertions.assertThat(callbackResponse.getConfirmationHeader()).isNotPresent();
+        Assert.assertTrue(callbackResponse.getConfirmationBody().isPresent());
```

Comment:
Same as earlier one. Use one library for assertThat to be consistent.

Reply:
Corrected Assert statements